.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea
 /vendor
-composer.lock
+/output
```

Comment:
composer.lock should not be part of the repo so it should be ignored.
output is also not really needed as it is not a default output directory.